pennylane/__init__.py

```diff
@@ -38,7 +38,6 @@
 
 * Optimization methods: :mod:`pennylane.optimize`
 * Configuration: :mod:`pennylane.Configuration <pennylane.configuration>`
-* Utilities: :mod:`pennylane.utils <pennylane.utils>`
```

Comment:
These are developer utilities used by the QNode (i.e., `unflatten`, `_flatten`) and should not be in the user-facing part of the documentation.